### PR TITLE
Migrate the CI/CD to GitHub Actions

### PR DESCRIPTION
**Pull request by s1lentq**
It looks like Travis-CI no longer has free plans for OSS.
So why not migrate to github-CI?

@WPMGPRoSToTeMa you need to create a secret key `API_KEY` in the repository settings to deploy the artifacts

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        if: |
+          github.event_name == 'release' &&
+          github.event.action == 'published' &&
+          startsWith(github.ref, 'refs/tags/')
+        with:
+          files: |
+            bin/linux32/SafeNameAndChat.so
+            bin/win32/SafeNameAndChat.dll
+        env:
+          GITHUB_TOKEN: ${{ secrets.API_KEY }}
```
Comment (WPMGPRoSToTeMa) [inline .github/workflows/build.yml]: As far as I can see this is no longer needed:
> The following `step.env` keys are allowed as a fallback but deprecated in favor of using inputs.
> https://github.com/softprops/action-gh-release#environment-variables

```suggestion
```

**Review by WPMGPRoSToTeMa — CHANGES_REQUESTED** [review]
> @WPMGPRoSToTeMa you need to create a secret key `API_KEY` in the repository settings to deploy the artifacts

As I can see this won't be needed, let me check later.

**Review by WPMGPRoSToTeMa — APPROVED** [review]
Thank you so much for this, LGTM :shipit:

**Comment by WPMGPRoSToTeMa** [comment]
I'll merge this about midnight MSK.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+
+    steps:
+    - name: Remove old artifacts
+      uses: c-hive/gha-remove-artifacts@v1
+      with:
+        age: '1 second'
+        skip-tags: true
+        skip-recent: 2
```
Comment (WPMGPRoSToTeMa) [inline .github/workflows/build.yml]: @s1lentq is cleanup really necessary? Are there any limitations on artifacts for public repositories?

**Review by WPMGPRoSToTeMa — APPROVED** [review]
I've tested this in a separate repository and it works great, thanks again! But I want to know a bit more information about the cleanup job.

File: .github/workflows/build.yml (hunk 2 above)
Comment (s1lentq) [reply .github/workflows/build.yml]: @WPMGPRoSToTeMa Most likely there is no limit for public repos

Comment (WPMGPRoSToTeMa) [reply .github/workflows/build.yml]: @s1lentq I'll remove it then.

**Review by WPMGPRoSToTeMa — APPROVED** [review]

**Comment by WPMGPRoSToTeMa** [comment]
Sorry for the delay, I didn't feel well this week.

**Pull request merged**